scripts/pythondistdeps.py

```diff
@@ -267,11 +277,17 @@ def normalize_name(name):
             # See https://bugzilla.redhat.com/show_bug.cgi?id=1791530
             normalized_name = normalize_name(dist.project_name)
 
+            # If we're processing an extras subpackage, check that the extras exists
+            if extras_subpackage and extras_subpackage not in dist.extras:
+                print(f"The package name contains an extras name `{extras_subpackage}` that was not found in the metadata.")
```

Comment:
Suggestion: Add something like: If this error is new and the extras were removed, consider removing the subpackage and obsoleting it from another subpackage.

Reply:
Amended, take a look.

Reply:
Resolved.